### PR TITLE
Extend motorbike profile and include profiles in builds

### PR DESCRIPTION
**Pull request by erikbosch**
This PR contains some signals relevant to motorbikes.
Some of them might be of general interest and may be moved to
"standard vss" later.

Signed-off-by: Erik Jaegervall <erik.jaegervall@se.bosch.com>

### CODE REVIEW

File: .github/workflows/buildcheck.yml [hunk 1]
```diff
@@ -25,9 +25,13 @@ jobs:
           python -V
           python -m pip --quiet --no-input install --upgrade pip 
           python -m pip --quiet --no-input install --upgrade pipenv wheel
+          pipenv --version
```
Comment (erikbosch) [inline .github/workflows/buildcheck.yml]: This file actually does not belong to this PR, shown here as github could not detect the rebase

File: Makefile [hunk 2]
```diff
@@ -37,6 +37,11 @@ csv:
 ddsidl:
 	${TOOLSDIR}/vspec2ddsidl.py -I ./spec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).idl
 
+# Verifies that supported overlay combinations are syntactically correct.
+overlays:
```
Comment (erikbosch) [inline Makefile:41]: Only indirectly related to the contribution - make sure that the overlay is syntactically correct

File: overlays/profiles/motorbike.vspec [hunk 3]
```diff
@@ -42,4 +43,63 @@ Vehicle.Cabin.Handlebar.GripHeating:
...
+Vehicle.IMU:
+    type: branch
+    description: Information related to the Inertial Measurement Unit to record angular rates and velocity-related information.
+
+Vehicle.IMU.IsOK:
+    type: sensor
+    datatype: boolean
+    description: Indicates if IMU is working. True = OK. False = Failure.
+
+# Next branch and signal can be removed when https://github.com/COVESA/vehicle_signal_specification/pull/478 is merged
```
Comment (erikbosch) [inline overlays/profiles/motorbike.vspec]: Added this comment, will update this PR when the other is merged

File: overlays/profiles/motorbike.vspec [hunk 4]
```diff
@@ -42,4 +43,63 @@ Vehicle.Cabin.Handlebar.GripHeating:
...
+    type: branch
+
+Vehicle.Powertrain.Transmission:
+    type: branch
+
+Vehicle.Powertrain.Transmission.IsClutchSwitchOperated:
+    type: sensor
+    datatype: boolean
+    description: Indicates if the Clutch switch is operated, so engine and transmission are decoupled. False = Clutch switch not operated. True = Clutch switch operated.
+    comment: True corresponds to that Vehicle.Powertrain.Transmission.ClutchEngagement is 0.0 or very close to 0.0.
```
Comment (erikbosch) [inline overlays/profiles/motorbike.vspec]: Some minor change here - I changed the signal name to comply with VSS "Is*" naming recommendation. But then It got a bit unsure on what we mean with this signal vs. Vehicle.Powertrain.Transmission.ClutchEngagement. Shall this be true as soon as you barely move the clutch switch, i.e. corresponding to ClutchEngagement is just below 100%, or shall it be true first when ClutchEngagement is almost 0?

File: Makefile [hunk 5]
```diff
@@ -37,6 +37,11 @@ csv:
...
+	${TOOLSDIR}/vspec2csv.py -I ./spec -o overlays/profiles/motorbike.vspec ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION)_motorbike.idl
```
Comment (SebastianSchildt) [inline Makefile]: Doesn't matter for the intended goal of checking "parsability" of the overlay, but test uses vspec2csv however  saves result  to .idl
ceterum censeo vspec2json ist the only tool that counts 😛

**Review by SebastianSchildt — APPROVED** [review]
In my opinion this can be proposed upstream.

It is anyway still small, and will grow. Let's start growing it :)

**Comment by erikbosch** [comment]
Replaced by https://github.com/COVESA/vehicle_signal_specification/pull/487
